panc/src/main/scripts/panlint

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding=utf8
+
+import re
+import argparse
+from glob import glob
+from colorama import Fore, Style, init as colorama_init
+from sys import stdout, exit as sys_exit
+from inspect import getmembers, ismethod
+from prettytable import PrettyTable
+
+RS_COMMENT = r'(?:#.*|@{.*})$'
+
+RE_STRING = re.compile(r'''('.*?'|".*?"(?<!\\"))''')
```

Comment:
can you give an example what you are trying to avoid with the negative lookbehind?